<?xml version="1.0"?>
<launch>

    <node name="cloud_express" pkg="pointcloud_usage" type="cloud_express"  output="screen" respawn="false" respawn_delay="0">
        <param name="load_file_name" value="/home/ericlab/OneDrive/pipe_0423/pattern_2/Angle45_1/Angle45_1pcd_pattern2_angle45.pcd"/>
        <param name="output_topic_name" value="shinya_2"/>
        <param name="parent_link" value="photoneo_center_optical_frame"/>
    </node>

</launch>